### PR TITLE
Implement image paragraph test

### PR DESCRIPTION
**Pull request by xendk**
Image paragraph test.

### CODE REVIEW

File: features/bootstrap/Traits/Paragraphs.php [hunk 1]
```diff
@@ -5,12 +5,12 @@
 trait Paragraphs
 {
     /**
-     * @When /^(?:|I )add a (.*) paragraph$/
+     * @When /^(?:|I )add an? (.*) paragraph$/
```
Comment (googletorp) [inline features/bootstrap/Traits/Paragraphs.php:8]: `an?` Guessing this is regex for optional n here, would be great with a comment explaining this, so you don't have to understand the Regex. I often need to look up the tech specs and easy to miss a detail

File: features/bootstrap/Page/Element/EditParagraphs.php [hunk 2]
```diff
@@ -25,5 +26,16 @@ public function add(string $type): void
...
         $this->waitForAjaxToFinish();
+
+        // Grab the first table, should be the table listing the paragraphs.
+        // This is to avoid getting tr elements from subforms.
+        $table = $this->find('xpath', '//table');
+
+        return count($table->findAll('xpath', '/tbody/tr'));
+    }
+
+    public function get(int $num): ?TraversableElement
```
Comment (googletorp) [inline features/bootstrap/Page/Element/EditParagraphs.php:37]: Not super happy about naming here, could be confusing later on where Drupal devs are used to ->get being able to fetch any data. Would be great with doctype for method as well.

File: features/paragraphs/image.feature [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  A paragraph that displays an image.
+
+  Background:
+    Given I am an editor
+
+  @api @javascript
+  Scenario: adding an image to an article
+    Given I am adding a new article
+    When I add an image paragraph
+    And I attach "duy-pham-Cecb0_8Hx-o-unsplash.jpg" as "Billede"
```
Comment (googletorp) [inline features/paragraphs/image.feature:11]: I'm a bit puzzled by the image name here - where does the image come from, is that a Behat thing? can you write anything or is there some magic?

Comment (xendk) [reply features/paragraphs/image.feature:11]: The "magic" is that it looks in files for a file with that name. It's simply the name of the test image.

File: features/bootstrap/Traits/Paragraphs.php (hunk 1 above)
Comment (xendk) [reply features/bootstrap/Traits/Paragraphs.php:8]: So you latch on to the `?` which is mostly the *first* character covered in any regex introduction, but didn't bat an eyelid over the non-capturing groups I've been using in every one? ;-)

File: features/bootstrap/Page/Element/EditParagraphs.php (hunk 2 above)
Comment (xendk) [reply features/bootstrap/Page/Element/EditParagraphs.php:37]: Well, as this *is* `EditParagraphs`, `getParagraph` seemed a bit redundant.

File: features/paragraphs/image.feature (hunk 3 above)
Comment (xendk) [reply features/paragraphs/image.feature:11]: On the other hand, Mink uses `find*` everywhere, so `get` is probably misnamed.

File: features/bootstrap/Page/Element/EditParagraphs.php (hunk 2 above)
Comment (xendk) [reply features/bootstrap/Page/Element/EditParagraphs.php:37]: On the other hand, Mink uses `find*` everywhere, so `get` is probably misnamed.

File: features/paragraphs/image.feature (hunk 3 above)
Comment (googletorp) [reply features/paragraphs/image.feature:11]: Shouldn't the file be added to git then? Maybe I'm missing something, but I can't see it

Comment (xendk) [reply features/paragraphs/image.feature:11]: It is.

https://github.com/reload/kk-basis/blob/master/features/files/duy-pham-Cecb0_8Hx-o-unsplash.jpg

**Pull request merged**